AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -3105,6 +3104,7 @@ animenewsnetwork.com#@##Adbanner
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=strategium.ru
 @@||yandex.ru/ads/system/context.js$xmlhttprequest,domain=sdamgia.ru
 timeanddate.com#@##ad-wrap2
+lite.liiingo.com#%#//scriptlet('prevent-setTimeout', 'ad_blocker_detector_modal')
```

Comment:
Before it was line 2903 and now 3107 (even below). Line 1635 would be better (for next ones, all required rules are available, so going to apply).